### DISCUSSION TITLE
Black screen and crash when attempting to play on a Meizu phone

### DISCUSSION THREAD

**Discussion by ljjsdsb** [ppy/osu#13768] [Q&A]
Meizu phone can't enter the game, black screen and flash back

**Comment by bdach** [comment]
I don't know what a "meizu phone" is, but your device might be incompatible with the game as things stand. Please attach logs from `Android/Data/sh.ppy.osulazer/logs`. And please state the actual make and model of the phone, with hardware specs too if possible.
